web/thesauruses/_meta/control_structures.json

```diff
@@ -52,6 +53,10 @@
       "name": "Ternary conditional",
       "code": ""
     },
+    "list_comprehension": {
```

Comment:
You technically did this correct, however, can you add `list_comprehension` in the same order as they are above? (so after the `foreach_loop`)